int32 strafe_X
int32 strafe_Y
int32 turn_X
int32 turn_Y
int32 descend
int32 ascend

bool lights_on
bool lights_off
bool lights_increase
bool lights_decrease

int32 arm_base
int32 arm_rot
bool arm_grip_open
bool arm_grip_close

bool free_roam
bool hold_position
